soh/soh/Enhancements/randomizer/3drando/custom_messages.cpp

```diff
@@ -280,7 +31,7 @@ constexpr std::array DungeonColors = {
     std::string SET_SPEED(uint8_t x) {
         return "\x7F\x10"s + char(x);
     }
-    std::string SKULLTULAS_DESTROYED() { return  "\x7F\x15"s; } //RANDOTODO just refernce the versions in CustomMessage
+    std::string SKULLTULAS_DESTROYED() { return  "\x7F\x15"s; } //RANDOTODO just reference the versions in CustomMessage
```

Comment:
```suggestion
    std::string SKULLTULAS_DESTROYED() { return  "\x7F\x15"s; }
```
we never reference this so it can probably be removed. In fact I think we may only be referencing `TWO_WAY_CHOICE`